scripts/utils.js

```diff
@@ -101,7 +101,11 @@ async function updateRemixVersion(packageName, nextVersion, successMessage) {
         config.devDependencies[`@remix-run/${pkg}`] = nextVersion;
       }
       if (config.peerDependencies?.[`@remix-run/${pkg}`]) {
-        config.peerDependencies[`@remix-run/${pkg}`] = nextVersion;
+        let isRelaxedPeerDep =
+          config.peerDependencies[`@remix-run/${pkg}`]?.startsWith("^");
+        config.peerDependencies[`@remix-run/${pkg}`] = `${
+          isRelaxedPeerDep ? "^" : ""
+        }${nextVersion}`;
```

Comment:
@chaance loosened this in #4736 for `@remix-run/dev`

Reply:
i'll defer to @chaance on this :)

Reply:
we still manually commit for nightly and experimental releases